src/hooks/useNotificationTypes.tsx

```diff
@@ -645,6 +647,46 @@ export const notificationTypes: NotificationTypeConfig[] = [
       const dispatch = useAppDispatch();
       const orders = useAppSelector(getSubaccountOrders, shallowEqual) ?? [];
 
+      return (orderClientId: string) => {
```

Comment:
I think u duplicated the orderClientId here and it's still on line 690 👀

Reply:
oops... okay let me update this